docs/general.md

```diff
@@ -534,4 +534,82 @@ add 8,r0,r1
 
 ## Bootloader
 
-Operator execution (transactions come in, get executed on the bootloader, state is suspended until new transaction shows up).
+The **Bootloader** is a system contract that orchestrates the construction and validation of new blocks, acting as a bridge between the EraVM and the external server to ensure accurate transaction processing.
+
+It validates and executes transactions, handles errors effectively, and integrates with EraVM for seamless block construction.
```

Comment:
I think we should mention why we need this to be a contract rather than part of the server functionality in the first place: to prove the execution of a batch we can't just prove the execution of the individual transactions, but rather we need to prove their sequencing, scheduling, loading and fees. This means we need to run all of this on top of the VM to make it provable, and the easiest way to do that is by writing it as a contract.

Reply:
I have added this information. Thank you!